# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/stressapptest/hotfix-reproducible.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Remove confusing and reproducible build breaking random build host logging.

--- stressapptest-1.0.11/src/sat.cc.vanilla	2025-04-30 16:51:22.336000000 +0300
+++ stressapptest-1.0.11/src/sat.cc	2025-04-30 16:51:44.810488615 +0300
@@ -553,7 +553,7 @@
 bool Sat::PrintVersion() {
   logprintf(1, "Stats: SAT revision %s, %d bit binary\n",
             kVersion, address_mode_);
-  logprintf(5, "Log: %s from %s\n", Timestamp(), BuildChangelist());
+  //logprintf(5, "Log: %s from %s\n", Timestamp(), BuildChangelist());
 
   return true;
 }
